spec/spec_helper_acceptance.rb

```diff
@@ -30,6 +30,13 @@
         on host, 'echo deb http://ftp.debian.org/debian jessie-backports main >> /etc/apt/sources.list'
         on host, 'apt-get update && apt-get install -y openhpid', acceptable_exit_codes: [0, 1, 100]
       end
+      # On Debian 9 "stretch", service state transitions (restart, stop) hang indefinitely and
+      # lead to test timeouts if there is a service unit of Type=notify involved.
+      # Use Type=simple as a workaround. See issue 455.
+      if fact('os.family') == 'Debian' && fact('os.release.major') == '9'
```

Comment:
Would it make sense to also check `host[:hypervisor] =~ %r{docker}`? There are alternative hypervisors (libvirt) which do use full blown images and it'd be nice to check the real service there.

Reply:
Yes, alright. I can make this change tomorrow the earliest, though.

Reply:
I've added a commit via the UI. Please have a look if you agree with it.